test/integration/setup.js

```diff
@@ -22,31 +22,28 @@ const request = require('request');
 
 // Run before all tests
 before(function(done) {
-	const that = this;
-
-	this.baseUrl = 'http://localhost:' + config.port + '/';
+	this.baseUrl = `http://localhost:${config.port}/`;
 	this.app = null;
 	this.last = {};
-	this.navigate = createNavigator(that.baseUrl, that.last);
+	this.navigate = createNavigator(this.baseUrl, this.last);
 
-	assertTestAppIsRunning(this.baseUrl, function() {
+	assertTestAppIsRunning(this.baseUrl, () => {
 		config.dbOnly = true;
-		app(config, function(error, initialisedApp) {
-			that.app = initialisedApp;
+		app(config, (error, initialisedApp) => {
+			this.app = initialisedApp;
```

Comment:
Same, why is `that` gone?

Reply:
Fat arrow functions keep the same block scope as where they are declared. So `this` still referes to the parent functions `this`. 
`function` on the other hand do not keep the same scope, which was why the `const that = this` was required.
As we're using fat-arrows now, theres no need for the `that`.